contrib/generate_profile.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+fdbdir=`pwd`/..
```

Comment:
This only works if you invoke this script from a particular directory.
Can we use some CMake variables?

Reply:
done